src/renderer/components/SessionList.tsx

```
@@ -118,6 +118,100 @@ function SessionContextMenu({
+				top,
+				opacity: ready ? 1 : 0,
+				...styles.container,
+				minWidth: '160px',
+			},
+			submenu: {
+				minWidth: '140px',
+				...submenuPositionStyles,
+			},
+		};
+	}, [
+		left,
```

Biome (2.4.4)

[error] This variable is used before its declaration.

(lint/correctness/noInvalidUseBeforeDeclaration)